liuliu/get-pull-and-update-pull-doc (#1647)
dolthub/docs: content/products/dolthub/api/database.md

}
```

{% swagger src="../../../.gitbook/assets/createpull.json" path="/{owner}/{database}/pulls" method="post" %}
[createpull.json](../../../.gitbook/assets/createpull.json)
{% endswagger %}

> ## Get details of a pull request

Reviewer: ```suggestion
## Get pull request details
```

> This API allows you to retrieve the details of a specific pull request in the `museum-collections`` database. In this example, we will retrieve the details of pull request #1.

Reviewer: ```suggestion
This API allows you to retrieve the details of a specific pull request in the `museum-collections` database. In this example, we will retrieve the details of pull request #1.
```

Include this `header` in your request.

```python
headers = {
    'authorization': '[api token you created]'
}
```

{% swagger src="../../../.gitbook/assets/getpull.json" path="/{owner}/{database}/pulls/{pull_id}" method="get" %}
[getpull.json](../../../.gitbook/assets/getpull.json)
{% endswagger %}

## Update a pull request

> This API allows you to update a pull request by providing the necessary information in the request body. You can update the title, description, and even close the pull request.

Reviewer: ```suggestion
This API allows you to update a pull request by providing the fields you want to update in the request body. You can update the title, description, and state (only closing a pull request is supported).
```
